src/index.js

```diff
@@ -28,30 +26,40 @@ const myRand = () => {
 for (let i = 0; i < 50; i++) {
   const delay = Math.random() + 's';
   const el = document.createElement('img')
-  el.src            = './images/star.svg'
-  el.className      = 'glitter-star'
-  el.style.top      = myRand() + '%'
-  el.style.left     = myRand() + '%'
-  el.style.animationDelay       = delay
-  el.style.msAnimationDelay     = delay
+  el.src = './images/star.svg'
+  el.className = 'glitter-star'
+  el.style.top = myRand() + '%'
+  el.style.left = myRand() + '%'
+  el.style.animationDelay = delay
+  el.style.msAnimationDelay = delay
   el.style.webkitAnimationDelay = delay
-  el.style.monAnimationDelay    = delay
+  el.style.monAnimationDelay = delay
   app.appendChild(el)
+}
 
 // logic below is for fetching the puzzles from the API. It will eventually be moved to the Round class probably
-const playGameBtn = document.querySelector('#start-game');
 
-playGameBtn.addEventListener('click', fetchPuzzles);
+generateRandomInfo();
+
+function generateRandomInfo() {
+  let possibleWordLengths = ['one', 'two', 'three', 'four'];
+  var puzzleWordLength = possibleWordLengths[Math.floor(Math.random() * possibleWordLengths.length)] + '_word_answers';
 
-let puzzleBank;
+  console.log(puzzleWordLength);
 
-function fetchPuzzles() {
-  event.preventDefault();
+  let randomNum = Math.floor((Math.random() * 23) + 0);
+  console.log(randomNum);
 
+  fetchPuzzles(randomNum, puzzleWordLength);
```

Comment:
How could you ensure the random number never gets repeated? Also, are you planning to call `fetchPuzzles()` every time a new round starts?